packages/api-generator/src/index.ts

```diff
@@ -18,7 +26,7 @@ const run = async () => {
   const pool = new Piscina({
     filename: './src/worker.js',
     niceIncrement: 10,
-    maxThreads: Math.max(1, Math.floor(Math.min(os.cpus().length / 2, os.freemem() / (1.1 * 1024 ** 3)))),
+    maxThreads: inspector.url() ? 1 : Math.max(1, Math.floor(Math.min(os.cpus().length / 2, os.freemem() / (1.1 * 1024 ** 3)))),
```

Comment:
is there any value in a comment block that explains what this essentially equates to?

Reply:
What, you can't read that?